### DISCUSSION TITLE
There is no way to add a fallback font?

### DISCUSSION THREAD

**Discussion by microhobby** [microsoft/terminal#15003] [Q&A] [answered]
I like to use old fonts that do not have the nerd fonts glyphs embedded on it, so in other terminals like Tabby I use the nerd fonts glyphs only as fallback. But I did not find a way to do it in the Windows Terminal. Someone knows if there is a workaround? Thanks

**Comment by zadjii-msft** [comment, marked as answer]
That's not currently possible, but tracked in #2664
